src/core/server/environment/write_pid_file.ts

```diff
@@ -18,6 +18,7 @@
  */
 
 import { unlinkSync as unlink } from 'fs';
+
```

Comment:
should we add the comment here?

Reply:
The ESLint rule only applies to code in client, common, or packages. On the server the bundle size isn't a concern.

Reply:
1.agree, but it adds inconsistency.
2. this change shouldn't have been added